server.js

```diff
@@ -3,36 +3,63 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import goldenGlobesData from './data/golden-globes.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 9000
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
+
+const Globe = new mongoose.model('Globe', {
+    year_film: Number,
+    year_award: Number,
+    ceremony: Number,
+    category: String,
+    nominee: String,
+    film: String,
+    win: {
+      type: Boolean,
+      required: true 
+    } 
+});
+
+
+if (process.env.RESET_DATABASE) {
+
+  const populateDatabase = async () => {
+    await Globe.deleteMany();
+
+    goldenGlobesData.forEach(item => {
+      const newGlobe = new Globe(item)
+      newGlobe.save();
+    })
+  }
+  populateDatabase();
+}
+
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello Internet')
+})
+
+app.get('/globes', async (req, res) => {
```

Comment:
Maybe the naming could be more specific here, to make it more RESTful. Not sure what globes is here? All nominations? :)